model_compression_toolkit/core/common/graph/base_node.py

```diff
@@ -96,6 +99,15 @@ def get_has_activation(self):
         """
         return self.has_activation
 
+    def get_has_positional_weights(self):
```

Comment:
change to `has_positional_weights` which is a `@property`, that checks whether there are positional weights (i.e. check for integers in the keys of the `weights` dictionary).